db/schema.rb

```diff
@@ -37,7 +37,8 @@
   end
 
   create_table "versions", force: :cascade do |t|
-    t.string "item_type", null: false
+    t.string "item_type"
+    t.string "{:null=>false}"
```

Comment:
This doesn't look like it should be here 🤔 Looks like it might be a problem with one of our earlier migrations using old Ruby syntax that's deprecated in our new Ruby version!

Reply:
I have fixed the other two issues, but I'm not sure how to fix that. Should I create a migration for that?

Reply:
For now we can just put this bit of the schema back how it was manually, and as long as you don't run all the migrations again from scratch it won't add this column again. Then we can fix the dodgy migration as a separate issue to this PR :)

Reply:
I think it's fixed :)

Reply:
Perfect, thank you!